### PR TITLE
Feature/refactor accountpage logic

### PR DESCRIPTION
**Pull request by caw442000**
# Description
Refactored code to use separate url for login and signup to rid of memory leak warning.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by ErikRodriguez-webdev** [comment]
Conflicts need to be resolved

File: src/App.js [hunk 1]
```diff
@@ -13,16 +13,26 @@ function App() {
...
-          <LandingPage />
-        </Route>
-        <PrivateRoute exact path="/dashboard" component={Dashboard} />
-        <Route exact path="/lesson">
-          <Lesson />
-        </Route>
-        <Route exact path="/account">
-          <Account />
-        </Route>
+        <Switch>
```
Comment (zeravenyoej) [inline src/App.js:16]: Switch is not imported

File: src/components/Account.js [hunk 2]
```diff
@@ -83,23 +85,30 @@ const useStyles = makeStyles((theme) => ({
...
+  const [value, setValue] = useState(props.value);
 
-  const handleChange = (event, newValue) => {
-    setValue(newValue);
+  const handleChange = (event, value) => {
+    // checks value and pushes url to tab without rerendering
+    if (value === 0) {
+      history.push("/account/signup");
+    }
+    if (value === 1) {
```
Comment (zeravenyoej) [inline src/components/Account.js:99]: be a tad cleaner if we made this an "else if"

**Comment by ErikRodriguez-webdev** [comment]
Looking Good!

**Pull request merged**